scripts/generator/graphql_generator/csharp/type.cs.erb

```diff
@@ -120,14 +120,19 @@ namespace <%= namespace %>.GraphQL {
     <% when 'INPUT_OBJECT' %>
         <%= docs_input_object(type) %>
         public class <%= type.classify_name %> : InputBase {
+
+            <% type.input_fields.each do |field| %>
+            public const string <%= field.name %>FieldKey = "<%= field.name %>";
```

Comment:
Yep this is made public because we need to reference these when handling `UserErrors` down the road